Description: Fix code to expect translation files in the standard folder
Author: Margarita Manterola <marga@debian.org>
Origin: vendor
Forwarded: no
Applied-Upstream: no
Last-Update: 2014-07-12
Index: cinnamon-screensaver/src/cinnamon-screensaver-command.c
===================================================================
--- cinnamon-screensaver.orig/src/cinnamon-screensaver-command.c	2014-04-26 21:59:38.592412058 +0200
+++ cinnamon-screensaver/src/cinnamon-screensaver-command.c	2014-07-12 14:14:52.208448035 +0200
@@ -351,7 +351,7 @@
         GError          *error;
 
 #ifdef ENABLE_NLS
-        bindtextdomain (GETTEXT_PACKAGE, "/usr/share/cinnamon/locale");
+        bindtextdomain (GETTEXT_PACKAGE, "/usr/share/locale");
 # ifdef HAVE_BIND_TEXTDOMAIN_CODESET
         bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
 # endif
Index: cinnamon-screensaver/src/cinnamon-screensaver-dialog.c
===================================================================
--- cinnamon-screensaver.orig/src/cinnamon-screensaver-dialog.c	2014-04-26 21:59:38.596412145 +0200
+++ cinnamon-screensaver/src/cinnamon-screensaver-dialog.c	2014-07-12 14:14:56.664557856 +0200
@@ -539,7 +539,7 @@
         char   *nolock_reason = NULL;
 
 #ifdef ENABLE_NLS
-        bindtextdomain (GETTEXT_PACKAGE, "/usr/share/cinnamon/locale");
+        bindtextdomain (GETTEXT_PACKAGE, "/usr/share/locale");
 # ifdef HAVE_BIND_TEXTDOMAIN_CODESET
         bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
 # endif
Index: cinnamon-screensaver/src/cinnamon-screensaver.c
===================================================================
--- cinnamon-screensaver.orig/src/cinnamon-screensaver.c	2014-04-26 21:59:38.596412145 +0200
+++ cinnamon-screensaver/src/cinnamon-screensaver.c	2014-07-12 14:15:08.112839784 +0200
@@ -60,7 +60,7 @@
         };
 
 #ifdef ENABLE_NLS
-        bindtextdomain (GETTEXT_PACKAGE, "/usr/share/cinnamon/locale");
+        bindtextdomain (GETTEXT_PACKAGE, "/usr/share/locale");
 # ifdef HAVE_BIND_TEXTDOMAIN_CODESET
         bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
 # endif
Index: cinnamon-screensaver/src/test-fade.c
===================================================================
--- cinnamon-screensaver.orig/src/test-fade.c	2014-04-26 21:59:38.600412233 +0200
+++ cinnamon-screensaver/src/test-fade.c	2014-07-12 14:14:37.360082309 +0200
@@ -79,7 +79,7 @@
         int     op, event, err;
 
 #ifdef ENABLE_NLS
-        bindtextdomain (GETTEXT_PACKAGE, "/usr/share/cinnamon/locale");
+        bindtextdomain (GETTEXT_PACKAGE, "/usr/share/locale");
 # ifdef HAVE_BIND_TEXTDOMAIN_CODESET
         bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
 # endif
Index: cinnamon-screensaver/src/test-passwd.c
===================================================================
--- cinnamon-screensaver.orig/src/test-passwd.c	2014-04-26 21:59:38.600412233 +0200
+++ cinnamon-screensaver/src/test-passwd.c	2014-07-12 14:14:48.376353625 +0200
@@ -216,7 +216,7 @@
         char           *nolock_reason = NULL;
 
 #ifdef ENABLE_NLS
-        bindtextdomain (GETTEXT_PACKAGE, "/usr/share/cinnamon/locale");
+        bindtextdomain (GETTEXT_PACKAGE, "/usr/share/locale");
 # ifdef HAVE_BIND_TEXTDOMAIN_CODESET
         bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
 # endif
Index: cinnamon-screensaver/src/test-watcher.c
===================================================================
--- cinnamon-screensaver.orig/src/test-watcher.c	2014-04-26 21:59:38.600412233 +0200
+++ cinnamon-screensaver/src/test-watcher.c	2014-07-12 14:14:43.696238324 +0200
@@ -80,7 +80,7 @@
         GError *error = NULL;
 
 #ifdef ENABLE_NLS
-        bindtextdomain (GETTEXT_PACKAGE, "/usr/share/cinnamon/locale");
+        bindtextdomain (GETTEXT_PACKAGE, "/usr/share/locale");
 # ifdef HAVE_BIND_TEXTDOMAIN_CODESET
         bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
 # endif
Index: cinnamon-screensaver/src/test-window.c
===================================================================
--- cinnamon-screensaver.orig/src/test-window.c	2014-04-26 21:59:38.600412233 +0200
+++ cinnamon-screensaver/src/test-window.c	2014-07-12 14:15:02.944712488 +0200
@@ -138,7 +138,7 @@
         GError *error = NULL;
 
 #ifdef ENABLE_NLS
-        bindtextdomain (GETTEXT_PACKAGE, "/usr/share/cinnamon/locale");
+        bindtextdomain (GETTEXT_PACKAGE, "/usr/share/locale");
 # ifdef HAVE_BIND_TEXTDOMAIN_CODESET
         bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
 # endif
